gumroad/src/gumroad-block/editor.scss

```diff
@@ -0,0 +1,15 @@
+/**
+ * #.# Editor Styles
+ *
+ * CSS for just Backend enqueued after style.scss
```

Comment:
not sure what front-end back-end mean in this context but seems very confusing to have the same class name have two very different visual styles based on priority of loading css files

Reply:
Sorry, This is a hold over from default, generated styles, I'll remove them for the PR.